utils/audit/Makefile

```diff
@@ -0,0 +1,94 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=audit
+PKG_VERSION:=2.8.5
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://people.redhat.com/sgrubb/audit
+PKG_HASH:=0e5d4103646e00f8d1981e1cd2faea7a2ae28e854c31a803e907a383c5e2ecb7
+
+PKG_MAINTAINER:=Thomas Petazzoni <thomas.petazzoni@bootlin.com>
+PKG_LICENSE:=GPL-2.0
+PKG_LICENSE_FILES:=COPYING
+PKG_CPE_ID:=cpe:/a:linux_audit_project:linux_audit
+
+PKG_FIXUP:=autoreconf
+
+PKG_USE_MIPS16:=0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/audit
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS:=+libaudit +@KERNEL_AUDIT
```

Comment:
```
Package audit is missing dependencies for the following libraries:
libauparse.so.0
```
Please also package `libauparse` and depend on it here.

Reply:
And please remove dependency on `@KERNEL_AUDIT` as that will have unwanted side-effects...

Reply:
The latest push packages libauparse, and the audit package now depends on it.